src/mongocxx/options/find.hpp

```diff
@@ -308,6 +330,7 @@ class MONGOCXX_API find {
    private:
     stdx::optional<bool> _allow_partial_results;
     stdx::optional<std::int32_t> _batch_size;
+    stdx::optional<bsoncxx::document::view_or_value> _collation;
```

Comment:
I'm more inclined to pimpl all options fields because we can expect new ones from the server over time and having half pimpl and half not seems inelegant.  That might be another thing for 2.0, however.